### PR TITLE
1660 pksim cli argument

### PR DESCRIPTION
**Pull request by msevestre**

### CODE REVIEW

File: src/PKSim.CLI/Properties/launchSettings.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
```
Comment (Yuri05) [inline src/PKSim.CLI/Properties/launchSettings.json:1]: do we need **launchSettings.json** on GitHub?

File: src/PKSim.CLI.Core/RunOptions/JsonRunOptions.cs [hunk 2]
```diff
@@ -11,6 +11,12 @@ public class JsonRunOptions : Notifier, IValidatable, IWithInputAndOutputFolders
       public IBusinessRuleSet Rules { get; } = new BusinessRuleSet();
       public SimulationExportMode ExportMode { get; set; }
 
+      /// <summary>
+      /// Indicates whether all outputs of the simulation should be exported or only the one actually selected in the simulation.
+      /// Default value is <c>TRUE</c>
+      /// </summary>
+      public bool RunForAllOutputs { get; set; } = true;
```
[the remark this answers is not in the document]
Comment (msevestre) [reply src/PKSim.CLI.Core/RunOptions/JsonRunOptions.cs:18]: Yes. We need to update here as well

File: src/PKSim.CLI/Properties/launchSettings.json (hunk 1 above)
Comment (msevestre) [reply src/PKSim.CLI/Properties/launchSettings.json:1]: Nope

**Review by Yuri05 — APPROVED** [review]

File: src/PKSim.CLI.Core/RunOptions/JsonRunOptions.cs (hunk 2 above)
[the remark this answers is not in the document]
Comment (msevestre) [reply src/PKSim.CLI.Core/RunOptions/JsonRunOptions.cs:18]: This is required for the InstallationValidator. QualificationRunner uses something else.
see https://github.com/Open-Systems-Pharmacology/QualificationRunner/blob/develop/src/QualificationRunner.Core/Services/QualificationEngine.cs#L112

Issue created for InstallationValidator already

[the remark this answers is not in the document]
Comment (Yuri05) [reply src/PKSim.CLI.Core/RunOptions/JsonRunOptions.cs:18]: ok

**Pull request merged**